CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Use default config name as service name and allow both service name and module name
```

Comment:
If you want to highlight that there are some new options for module configuration, sure. Lets focus on that.

The changelog here is beta1 -> beta2, so there is definitely nothing breaking.